app/assets/stylesheets/organisms/_document-preview.scss

```diff
@@ -1,36 +1,28 @@
-.document-previewer {
-  display: inline-block;
-  margin-bottom: 1em;
+.doc-preview-container {
```

Comment:
one think I noticed on GCF is that we used `doc` and `document` interchangeably and I never knew which one to use where. What do you think about attempting to use one or the other in this project so there is less guesswork?